test/sysinfo.jl

```diff
@@ -7,6 +7,8 @@ sprint(Base.Sys.cpu_summary)
 @test Base.Sys.uptime() > 0
 Base.Sys.loadavg()
 
+@test !("fakename" ∈ keys(ENV))
```

Comment:
Maybe something with an = sign in it, since that's not a valid environmental variable name (at least on POSIX).